doc/shared/sunstepper/SUNStepper_Description.rst

```diff
@@ -346,3 +374,9 @@ abstract base class.
    This type represents a function with the signature of
    :c:func:`SUNStepper_SetForcing`.
 
+
+.. c:type:: SUNErrCode (*SUNStepperDestroyFn)(SUNStepper stepper)
```

Comment:
```suggestion
.. c:type:: SUNErrCode (*SUNStepperDestroyFn)(SUNStepper* stepper)
```

Reply:
This user-specified function is only responsible for freeing the content. It can' be responsible for freeing ops and the stepper itself since those details are not public. To reduce the risk of users freeing the wrong things, I think we should keep the argument as `SUNStepper stepper`.

Reply:
Ah, yes this is a bit different since the definition of the `SUNStepper` is private. I think this is fine as is then.